.circleci/config.yml

```diff
@@ -93,6 +93,10 @@ commands:
         description: "Name of CircleCi project environment variable that
           holds session secret, a required application variable"
         type: env_var_name
+      jwt_secret:
+        description: "Name of CircleCi project environment variable that
+          holds the JWT secret, a required application variable"
```

Comment:
It might be nice for future engineers if we explained where we are using the JWT

Reply:
updated here: c746bad0